### DISCUSSION TITLE
Send GIFT and Setting the label Handling Methods ?

### DISCUSSION THREAD

**Discussion by CapLuo** [devlikeapro/waha#358] [Q&A]
Hi, experts and authors,

How can I send gift image to other and How to create label for contracts？

Please help

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by allburov** [comment]
Hi! Do you mean GIF (like image video)? 

For GIFs - you need to convert it to video locally and after that send as video
 https://waha.devlike.pro/docs/how-to/send-messages/#send-video-imagesversionspluspng
It's the way how WhatsApp clients handles it

```json
POST /api/sendVideo
{
  "session": "default",
  "chatId": "11111111111@c.us",
  "caption": "Watch this video!",
  "file": {
    "mimetype": "video/mp4",
    "filename": "video.mp4",
    "url": "https://github.com/devlikeapro/whatsapp-http-api/raw/core/examples/video.mp4"
  }
}
```




[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by allburov** [comment]
We'll add full label support a bit later in https://github.com/devlikeapro/waha/issues/318

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
